lib_comfyui/custom_extension_injector.py

```diff
@@ -0,0 +1,58 @@
+import os
+import inspect
+import ast
+import textwrap
+
+webui_custom_nodes_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), 'comfyui_custom_nodes')
+webui_custom_scripts_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), 'comfyui_custom_scripts')
+
+
+def register_webui_extensions():
+    register_custom_nodes()
+    register_custom_scripts()
+
+
+def register_custom_nodes():
+    from folder_paths import add_model_folder_path
+    add_model_folder_path('custom_nodes', webui_custom_nodes_path)
+
+
+def register_custom_scripts():
+    import server
+
+    m = ast.parse(source(server.PromptServer))
+    patch_prompt_server_init(m)
+    patch_prompt_server_add_routes(m)

```

Comment:
Small detail, maybe we should call these 2 with `m.body[0]` instead of just `m` to pay respect to demeter.

Reply:
also we should use a full name instead of just `m`, for example `module`

Reply:
I think we should consider adding a couple type annotations. Here it should be `ast.Module`